contribs/docker/wazo-auth-mock/wazo-auth-mock.py

```diff
@@ -408,11 +410,19 @@ def token_get(token: str) -> Response | tuple[str, int]:
 
 def _valid_acl(token_id: str) -> bool:
     required_acl = request.args.get('scope')
-    if required_acl and 'acl' in valid_tokens[token_id]:
-        if required_acl in valid_tokens[token_id]['acl']:
-            return True
-        return False
-    return True
+    if not required_acl:
+        logger.debug('Valid ACL: no required access')
+        return True
+    if 'acl' not in valid_tokens[token_id]:
```

Comment:
To avoid accessing the same keys multiple times you could use the assignment expression. Not that it is a big worry.
```python
if 'acl not in (token := valid_tokens[token_id]):
    return True
if ACCEPT_ALL in (acls := token['acl']):
    return True
if required_acl in acls:
```